src/BasicInputs/BasicDateInput.js

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import 'date-fns';
+import DateFnsUtils from '@date-io/date-fns';
+import { withStyles } from '@material-ui/core';
+import {
+  MuiPickersUtilsProvider,
+  KeyboardDatePicker,
+} from '@material-ui/pickers';
+
+const useStyles = theme => ({
+  root: {
+    flexGrow: 1
+  }
```

Comment:
classes.root is not used in this file, we can remove "root" from the styles dictionary